meta_review/migrations/0002_auto_20180707_0305.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.14 on 2018-07-07 03:05
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('meta_review', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RenameField(
```

Comment:
this is should be a separate commit; and probably better to be a separate PR so it goes in straight away and your final PR doesnt include this.

Reply:
Separate PR created: https://github.com/coala/community/pull/166